relay-server/src/envelope.rs

```diff
@@ -96,10 +96,6 @@ pub enum ItemType {
     UnrealReport,
     /// User feedback encoded as JSON.
     UserReport,
-    /// Session update data.
-    Session,
-    /// Aggregated session data.
-    Sessions,
```

Comment:
Relay now converts session items from SDKs to metrics, but we still need to accept these incoming item types.